src/services/catalog.service.ts

```diff
@@ -127,7 +127,7 @@ export class ServiceItem implements IServiceItem {
   }
 
   private getDescription() {
-    return _.get(this.resource, 'osbMetadata.description', '');
+    return _.get(this.resource, 'description', '');
```

Comment:
Thought of this after the merge, should I update the mock data and move description from under osbMetadata?